gridmet_split_script.py

```diff
@@ -0,0 +1,219 @@
+# Libraries
+import pickle
+import time
+import geopandas as gpd
+import geopandas.geodataframe
+import grd2shp_xagg
+import xagg as xa
+import xarray as xr
+import os
+import datetime
+
+# Variables
+
+## official list of variables needed for drb-inland-salinity model
+data_vars_shrt_all = ['tmmx', 'tmmn', 'pr', 'srad', 'vs','rmax','rmin','sph']
+## drb catchment polygons
+gdf_nhru02_path = './data/nhru_02/nhru_02.shp'
+gdf = gpd.read_file(gdf_nhru02_path)
+## date range
+start_date = '1979-01-01'
+end_date = '2021-01-01'
+## lat lon
+lon_min, lat_min, lon_max, lat_max = gdf.total_bounds
+## output folder
+output_path = './data/'
+
+
+def get_gridmet_datasets(variable, start_date, end_date, polygon_as_bbox = None, lon_min = None, lat_min = None, lon_max = None, lat_max = None):
```

Comment:
I think the parameter `polygon_as_bbox` is a bit confusing. Maybe it would be better to say `polygon_for_bbox`. `polygon_as_bbox` makes me think that I should pass a polygon of the bbox when actually I just need to pass in my polygon.